tests/storage/snapshot.go

```diff
@@ -1053,7 +1053,7 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 
 				errStr := fmt.Sprintf("VolumeSnapshots (%s) missing", *vb.VolumeSnapshotName)
 				Expect(snapshot.Status.Error).ToNot(BeNil())
-				Expect(snapshot.Status.Error.Message).ToNot(Equal(errStr))
+				Expect(snapshot.Status.Error.Message).ToNot(HaveValue(Equal(errStr)))
```

Comment:
This is now failing in e2e.

The reason it was passed until now is that `snapshot.Status.Error.Message` is a pointer, compared to a string. This is always true.

With the fix, it seems that the value actually does equal to the value of the `errStr` variable.

@mhenriks - could you please take a look? How should we address it? Should we revert the assertion logic?

Reply:
Yeah I believe the original intention was that the error is == errStr, mhenriks will keep me honest